src/actions/actionToggleViewMode.tsx

```diff
@@ -0,0 +1,21 @@
+import { CODES, KEYS } from "../keys";
+import { register } from "./register";
+import { trackEvent } from "../analytics";
+
+export const actionToggleViewMode = register({
+  name: "viewMode",
+  perform(elements, appState) {
+    trackEvent("contextmenu", "viewMode");
```

Comment:
```suggestion
    trackEvent("view", "mode", "view");
```

(in case we will track, grid, zen, etc..